Profiler_panel/manifest.json

```diff
@@ -5,13 +5,14 @@
   "manifest_version": 3,
   "devtools_page": "devtools.html",
   "permissions": [
+    "storage",
     "http://*/*",
     "https://*/*",
     "chrome-extension://*/*",
```

Comment:
unrelated but do we need these permissions? chrome-extension etc

Reply:
This was added previously as part of the coverage feature